File: service/api/src/main/java/org/delivery/api/exception/model/ErrorResponse.java
package org.delivery.api.exception.model;

import org.delivery.api.common.api.Result;

public record ErrorResponse(
	String fieldName,
	String message
) {

	@Override
	public String toString() {
		return "ErrorResponse{" +
			"fieldName='" + fieldName + '\'' +
			", message='" + message + '\'' +
			'}';
	}

}

File: service/common/src/main/kotlin/org/delivery/common/exception/model/ErrorResponse.kt
package org.delivery.common.exception.model

data class ErrorResponse(
    val fieldName: String,
    val message: String
) {

    override fun toString(): String {
        return "ErrorResponse{" +
                "fieldName='" + fieldName + '\'' +
                ", message='" + message + '\'' +
                '}'
    }

}